src/views/SystemIntake/index.tsx

```diff
@@ -61,135 +68,121 @@ export const SystemIntake = () => {
   };
 
   const dispatchSave = () => {
-    if (id && formikRef.current.dirty) {
-      dispatch(saveSystemIntake({ formData: formikRef.current.values, id }));
+    if (formikRef.current.dirty) {
+      dispatch(saveSystemIntake(formikRef.current.values));
+      if (window.location.pathname === '/system/new') {
```

Comment:
After this save, we can probably set `formikRef.current.dirty = false`.

Reply:
```suggestion
      if (!match.params.systemId) {
```

Reply:
I don't think we should be changing values in the formikRef, right? My ideal scenario is that we could check "Are the current values different from the last saved values" rather than "Are the current values different from the values when the page loaded?"

Reply:
I agree with Isaac here around the changed values saved since auto save may have changed the status.

Reply:
My thinking is that after a form is saved, the form should no longer be `dirty`. We can do that by either manually changing it on the `ref` or by resetting the form. I have don't know if mutating the `ref` is bad practice, but doing something like `formikRef.current.resetForm({ values: formikRef.current.values })` might work not sure about that though.